canonical-web-and-design/maas-ui
# refactor(ui): update the network actions to be reusable

## Done

- Update the network actions component to be reusable by devices.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the network tab for a machine.
- Check that you see buttons for add interfaces, bonds etc. just as they were before.
- The buttons should open the forms (just as they did before).

## Fixes

Fixes: canonical-web-and-design/app-tribe#566.
